testing/dart/encoding_test.dart

```diff
@@ -114,7 +114,7 @@ class GrayscaleImage {
   GrayscaleImage._();
 
   static Future<Image> load() async {
-    final Uint8List bytes = await readFile('4x4.png');
+    final Uint8List bytes = await readFile('2x2.png');
```

Comment:
The file below seems unchanged. Did you forget to stage it?

Reply:
Hmm, the existing assertions seem unchanged. Maybe the file was incorrectly named before?

Reply:
The file is actually a 2x2 png, not 4x4. I renamed the file and updated the strings.